### PR TITLE
Oasis proxy

### PR DESCRIPTION
**Pull request by ethanbennett**
Integrates the Oasis Direct proxy with the testchain deployment script

Dai.js PR: https://github.com/makerdao/dai.js/pull/53

### CODE REVIEW

**Review by d-xo — APPROVED** [review]

File: out/DSAuth.abi
Comment (levity) [inline out/DSAuth.abi]: this shouldn't be here

**Review by levity — APPROVED** [review]
LGTM once you remove the stray ABI

**Comment by levity** [comment]
But please don't merge until the corresponding dai.js branch is passing tests.

**Review by konstantinzolotarev — APPROVED** [review]
LGTM

**Pull request merged**
